fw-child/resources/scss/_variables.scss

```diff
@@ -1604,10 +1604,10 @@ $figure-caption-color:              var(--#{$prefix}secondary-color) !default;
 // Breadcrumbs
 
 // scss-docs-start breadcrumb-variables
-$breadcrumb-font-size:              1.25rem;
+$breadcrumb-font-size:              1rem;
```

Comment:
The breadcrumb size should be reduced only on smaller screens. On bigger screens, the original size of `1.25rem` was more appropriate.

Suggestion: add custom rules for `.breadcrumb` in `_vendor.scss`.

Reply:
@xfrenette-crim rule for desktop breadcrumbs was added. Thanks!